cmake/projects/PNG/hunter.cmake

```diff
@@ -87,6 +87,28 @@ hunter_add_version(
     fbf7c49f144a32567a1b63cb658a2a4584a091fa
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    PNG
+    VERSION
+    "1.6.26-p6"
+    URL
+    "https://github.com/hunter-packages/libpng/archive/v1.6.26-p6.tar.gz"
+    SHA1
+    dee61a3f48ed2d436a4e7fb8cdaf3cabaa3e604e
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    PNG
+    VERSION
+    "1.6.36-p3"
```

Comment:
Have you tested it? If yes, please update `hunter_default_version`, otherwise please remove this.

Reply:
I only tested it locally on Linux with gcc and the respective mingw cross compiler. That is why I left the default at 1.6.26, but since there is already a package for 1.6.36 I thought adding it would allow hunter users to upgrade manually and see if it works for their toolchain. But if you require CI testing I will remove it.

Reply:
I think there are issues with 1.36 version, see:
- https://github.com/hunter-packages/libpng/pull/5

So it's better to remove this version until they are fixed.

You can always do:
```cmake
hunter_config(
    PNG
    VERSION "1.6.36-p3"
    URL"https://github.com/hunter-packages/libpng/archive/v1.6.36-p3.tar.gz"
    SHA1 3ad1207c649eb17d840d75bef048fe9ee12fa00a
)
```
in your local config. See documentation:
- https://docs.hunter.sh/en/latest/reference/user-modules/hunter_config.html

Reply:
> I think there are issues with 1.36 version, see:
> 
> * [hunter-packages/libpng#5](https://github.com/hunter-packages/libpng/pull/5)
> 
> So it's better to remove this version until they are fixed.

Okay, I will remove it from the PR.

> You can always do:
> 
> ```cmake
> hunter_config(
>     PNG
>     VERSION "1.6.36-p3"
>     URL"https://github.com/hunter-packages/libpng/archive/v1.6.36-p3.tar.gz"
>     SHA1 3ad1207c649eb17d840d75bef048fe9ee12fa00a
> )
> ```
> 
> in your local config.

Ah right, I forgot that you could also specify the URL, thank you!